doc/source/whatsnew/v1.1.0.rst

```diff
@@ -524,39 +466,6 @@ those integer keys is not present in the first level of the index (:issue:`33539
 
     left_df.merge(right_df, on=['animal', 'max_speed'], how="right")
 
-.. ---------------------------------------------------------------------------
```

Comment:
where did this go?

Reply:
https://github.com/pandas-dev/pandas/pull/34865/files#diff-e763086372257ed7ca944d8bc5042a04R812